_data/radar.yaml

```diff
@@ -1,13 +1,171 @@
+# ring 0 = ADOPT
+# ring 1 = TRIAL
+# ring 2 = ASSESS
+# ring 3 = HOLD
+
 version: 2019-11
 quadrants:
   - name: Security
-    technologies: []
+    technologies:
+
+      - name: Vault
+        link: https://www.vaultproject.io/
+        ring: 0
+
+      - name: Open Policy Agent
+        link: https://www.openpolicyagent.org/
+        ring: 1
 
   - name: CI/CD
-    technologies: []
+    technologies:
+
+      - name: Terraform
+        link: https://terraform.io
+        ring: 0
+
+      - name: Helm
+        link: https://helm.sh
+        ring: 0
+
+      - name: Jenkins X
+        link: https://jenkins-x.io/
+        ring: 1
+
+      - name: Flux
+        link: github.com/fluxcd/flux
+        ring: 1
+
+      - name: Tekton
+        link: https://tekton.dev
+        ring: 2
+
+      - name: GitHub Actions
+        link: https://github.com/features/actions
+        ring: 2
+
+      - name: Argo
+        link: https://argoproj.github.io/
+        ring: 2
+
+      - name: Bamboo
+        link: atlassian.com/software/bamboo
+        ring: 4
+
+      - name: Octopus Deploy
+        link: https://octopus.com/
+        ring: 3
+
+      - name: Travis CI
+        link: https://travis-ci.com/
+        ring: 3
+
+      - name: Jenkins
+        link: https://jenkins.io/
+        ring: 4
+
+      - name: TeamCity
+        link: https://www.jetbrains.com/teamcity/
+        ring: 3
+
+  - name: Orchestration & Management
+    technologies:
+
+      - name: Kubernetes
+        link: https://kubernetes.io
+        ring: 0
+
+      - name: gRPC
+        link: https://grpc.io/
+        ring: 1
+
+      - name: Envoy
+        link: https://www.envoyproxy.io/
+        ring: 1
+
+      - name: Kong
+        link: https://konghq.com/
+        ring: 1
+
+      - name: Ambassador
+        link: https://www.getambassador.io/
+        ring: 2
+
+      - name: Gloo
+        link: https://www.solo.io/
+        ring: 2
+
+      - name: Knative
+        link: https://knative.dev
+        ring: 2
+
+      - name: Docker Swarm
+        ring: 3
+
+      - name: Docker EE
+        ring: 3
+
+      - name: Nomad
+        link: https://www.nomadproject.io/
+        ring: 3
+
+      - name: Consul
+        link: https://www.consul.io/
+        ring: 3
+
+      - name: Sceptre
+        link: https://sceptre.cloudreach.com
+        ring: 3
+
+      - name: Puppet
+        link: https://puppet.com/
+        ring: 2
+
+  - name: Observability & Analysis
+    technologies:
+
+      - name: Prometheus
+        link: https://prometheus.io
+        ring: 0
+
+      - name: Grafana
+        link: https://grafana.com/
+        ring: 0
+
+      - name: Fluentd
+        link: https://www.fluentd.org/
+        ring: 0
+
+      - name: cert-manager
+        link: http://docs.cert-manager.io/en/latest/
+        ring: 0
+
+      - name: Istio
```

Comment:
Why is this placed under Observability & Analysis? Should it be below security?

Reply:
While Istio certainly has some security components to it, it is not primarily a security tool. Some tools may fall under multiple quadrants and we try to place them in their primary one. For guidance you can use the CNCF landscape for reference.

I don't have a particular strong opinion and if public's mindset of Istio is as a security tool we should move it. However there are multiple other service mesh products that are not as security focused as Istio hence why I chose to put it in Observability & Analysis.